embedded-graphics/src/lib.rs

```diff
@@ -1,5 +1,12 @@
-//! This crate aims to make drawing 2D graphics primitives super easy. It currently supports the
-//! following built in items:
+//! Embedded-graphics is a 2D graphics library that is focused on memory constrained embedded devices.
+//!
+//! A core goal of embedded-graphics is to draw graphics without using any buffers; the crate is
+//! `no_std` compatible and works without a dynamic memory allocator, and without pre-allocating
+//! large chunks of memory. To achieve this, it takes an `Iterator` based approach, where pixel
+//! values and positions are calculated on the fly, with the minimum of saved state. This allows the
```

Comment:
IMO we should replace "pixel values" by "pixel colors". You normally work with specific types that represent colors and not a raw value.